### PR TITLE
Make ansible unbuffered for better experience while logging

### PR DESCRIPTION
**Pull request by nedbat**

### CODE REVIEW

File: util/vagrant/upgrade.sh [hunk 1]
```diff
@@ -296,11 +293,8 @@ echo "ora2_version: $TARGET" >> vars.yml
 echo "certs_version: $TARGET" >> vars.yml
 echo "forum_version: $TARGET" >> vars.yml
 echo "xqueue_version: $TARGET" >> vars.yml
-sudo ansible-playbook \
-    --inventory-file=localhost, \
-    --connection=local \
+$ANSIBLE_PLAYBOOK \
```
Comment (nedbat) [inline util/vagrant/upgrade.sh:299]: This use of the macro will swap the `--extra-vars` switch and the `$SERVER_VARS` variable: used to be that `--extra-vars` was first, now `$SERVER_VARS` is first.  Does the ordering of those two matter?

**Comment by nedbat** [comment]
@edx/devops This keeps ansible output flowing with the progress of the playbook, and simplifies the handful of playbook invocations that we have.

File: util/vagrant/upgrade.sh (hunk 1 above)
Comment (feanil) [inline util/vagrant/upgrade.sh:299]: The ordering does matter as ansible will override earlier vars with those added from later files.  So this likely breaks people that are using values in server vars to override the defaults we provide.

Comment (nedbat) [inline util/vagrant/upgrade.sh:299]: OK, i've moved `$SERVER_VARS` outside of the `$ANSIBLE_PLAYBOOK` variable so as not to disrupt the ordering.

**Comment by nedbat** [comment]
@edx/devops now ok?

**Comment by feanil** [comment]
:+1:

**Pull request merged**